api/tests/routes/pro/post_collective_offers_template_test.py

```diff
@@ -146,6 +146,23 @@ def test_create_collective_offer_template(
         assert offer.end == template_end
         assert offer.author == user
 
+    def test_empty_email(self, pro_client, payload, venue):
+        data = {
+            **payload,
+            "contactEmail": None,
+        }
```

Comment:
(détail) il n'est peut-être pas nécessaire de créer un nouveau `dict` ici.

```suggestion
        payload["contactEmail"] = None
```